ApiReview/API.Polly.Core/NoDocs/Polly.CircuitBreaker/SimpleCircuitBreakerStrategyOptions.TResult.cs

```diff
@@ -0,0 +1,13 @@
+// Assembly 'Polly.Core'
+
+using System.ComponentModel.DataAnnotations;
+using System.Runtime.CompilerServices;
+
+namespace Polly.CircuitBreaker;
+
+public class SimpleCircuitBreakerStrategyOptions<TResult> : CircuitBreakerStrategyOptions<TResult>
+{
+    [Range(1, int.MaxValue)]
+    public int FailureThreshold { get; set; }
```

Comment:
To avoid confusion with the ratio based threshold from the Advanced circuit breaker, can we name this property in a similar manner with Polly V7 argument (i.e. handledEventsAllowedBeforeBreaking)?
Or we could name this `FailureCountThreshold` and the other `FailureRateThreshold` , wdyt?

Reply:
I like `FailureCountThreshold`, it's more expressive.

Reply:
I agree that we should always aim to be more explicit with our property names. I also like your proposed names for these.